instructions/index.html

```diff
@@ -31,10 +31,20 @@ <h3>Experience</h3>
   </ul></p>
 </section>
 
+<section id="skills">
+  <h2>Skills</h2>
+  <ul></ul>
+</section>
+
+
 <section>
   <h3>Connect</h3>
   <a href="https://github.com/roselynn411">Github Account</a>
   <p>My email is roselynnfrempong@gmail.com. My Instagram is rose.lynnn</p>
+  <footer id="copyright"> 
+    <p></p>
```

Comment:
Since we're creating the paragraph element here: `const copyright = document.createElement("p");` in the javascript file I don't think we need this other paragraph element.